blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubEnterpriseScm.java

```diff
@@ -28,6 +30,11 @@ public GithubEnterpriseScm(Reachable parent) {
         return super.getUri()+DEFAULT_ENTERPRISE_API_SUFFIX;
     }
 
+    @Override
+    public String getCredentialDomainName() {
+        return DOMAIN_NAME;
```

Comment:
better to keep these id's in their own domain, for example GitHub enterprise is in it's own domain so is consistent that way.